src/status_im/ui/screens/chat/extensions/views.cljs

```diff
@@ -37,7 +37,7 @@
                                   :height    panel-height
                                   :transform [{:translateY bottom-anim-value}]
                                   :opacity   alpha-value}}
-     [react/view
+     [react/view {:style {:flex-direction :row}}
```

Comment:
why row ? @errorists

Reply:
@errorists can tell if that's a good idea or not, basically I kept finding it super weird to have only 2 commands that are aligned vertically instead of horizontally, in the designs they are on top of each other but there is also more commands. So I thought we should align them like this until there is more